### ISSUE TITLE
refinery cms spike investigation

### ISSUE THREAD

**Issue by saracarl** [benwbrum/fromthepage#386]
timebox this investigation to 2 days

-- Get the gem installed and see if you can make it work with existing users
-- any gem upgrades needed?

-- do we need to change our devise/user model to work with current version of refinery?
-- does refinery have a blog plugin/component?
-- static pages
-- futures -- customers.fromthepage.com -- refinery content page that is limited to them??

**Comment by tbl73** [comment]
There are definitely a great many gem updates - I had to upgrade Rails to 4.2.6, update devise, use the latest rmagick (although I am anyway), specify a version of will_paginate (the newest works, but it wasn't happy until I specified it).  Also the refinery gem has a ton of dependencies - installing it also installed: zilch-authorisation, decorators, jquery-ui-rails, font-awesome-sass, awesome-nested-set, rails-i18n, routing-filter, refinerycms-i18n, refinerycms-core, globalize, dragonfly, acts_as_indexed, refinerycms-resources, diffy, speakingurl-rails, babosa, friendly_id, refinerycms-pages, and refinerycms-images.

I got it installed and got the server running again, but it's really not working all that well at this point.  I can't connect from outside the localhost anymore.  It's not finding routes that are clearly there.  And it doesn't look good, although it's likely that some of that can  be overridden.
